resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/service/ResourceServicePersistenceProxy.java

```diff
@@ -78,7 +77,7 @@ public ResourceServicePersistenceProxy(final PersistenceLayer persistence, final
      * @return a {@link Source} of {@link LeafResource}s associated with the resourceId
      */
     public Source<AuditableResourceResponse, NotUsed> getResourcesById(final ResourceRequest request) {
-        LOGGER.info(STORE);
+        LOGGER.info("Trying to get resourcesById for request {} from persistence", request);
```

Comment:
Is there not a way we can leave this as a constant to be used by all the get methods? Even if it's a case of passing in some relevant values for each time it's used.
"Trying to get 'methodName' for request {} and 'value to be used' from persistence"
I'm not sure that the entire request needs to be in the log message.

Reply:
They all seem reasonably different (method, name of second thing e.g. id/type/format)
I will move the request to debug level with a static message